### PR TITLE
Extra Series article toppers

### PR DESCRIPTION
**Pull request by adgad**
This escalated quickly. Along with adding the series header topper, I've also started to refactor toppers so they all (topper + normal headers) are the same. (as well as deleting some unused code i've come across along the way)

- [x] Add "extra" series article topper
- [x] Rename article-topper => topper
- [x] Remove `topper utils` and make everything use the same thing
- [x] Make the normal heading a topper
- [x] "Fix" the 150vw thing on offset toppers
- [x] Branding and headshots (new topper theme?)
- [x] Delete all the old header code

### CODE REVIEW

File: client/components/article/topper/themes/_full-bleed-offset.scss [hunk 1]
```diff
@@ -36,6 +38,17 @@
 	.article-topper__offset-container {
 		margin: -90px -10px 0;
 		padding: 0 10px;
+		position: relative;
+		&:before {
+			content: '';
+			position: absolute;
+			left: 50%;
+			width: 150vw; //go extra wide to ensure its always covered
```
Comment (TheoLeanse) [inline client/components/article/topper/themes/_full-bleed-offset.scss]: as we just discussed this might cause some unwanted horizontal scrolling

**Comment by TheoLeanse** [comment]
+1

File: client/components/topper/_main.scss [hunk 2]
```diff
@@ -1,18 +1,70 @@
...
+	font-size: round(18px * 0.9);
+	line-height: 1.4;
+
+	.o-typography--loaded-sans & {
+		font-size: 18px;
+		line-height: 1.2;
+	}
+
+	@include oGridRespondTo(L) {
+		font-size: round(22px * 0.9);
```
Comment (gvonkoss) [inline client/components/topper/_main.scss]: I believe Oli asked for all standfirsts to be 22px on [this card](https://trello.com/c/RsnAwNro/137-standfirst-on-topper-was-22px-but-has-dropped-to-18px)

File: client/components/topper/_main.scss [hunk 3]
```diff
@@ -105,48 +120,44 @@
...
 	font-weight: 400;
 }
 
-.article-topper__tags {
+.topper__tags {
 	display: flex;
 	padding-bottom: 5px;
 
-	@include oGridRespondTo(M) {
-		width: 45%;
```
Comment (gvonkoss) [inline client/components/topper/_main.scss]: Do the tags still behave the same without this breakpoint? (same for split left right and center?)

File: client/components/topper/themes/_full-bleed-image-left.scss [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
+.topper--full-bleed-image-left {
+
+	padding: 0;
+
 	@include alignText(left);
 	@include oGridRespondTo(S) {
 		@include overlayText;
 		@include fullBleedImage;
-		.article-topper__tags {
-			width: inherit;
```
Comment (gvonkoss) [inline client/components/topper/themes/_full-bleed-image-left.scss]: definitely maintains the same behaviour across viewports w/o this? (for all themes)

File: client/components/topper/themes/_full-bleed-offset.scss [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+				left: 70.666%; //These are magic numbers to compensate for grid offset/widths
+
+			}
+		}
+
+	}
+
+	@include oGridRespondTo($from: M) {
+		.topper__offset-container {
+			margin-left: -20px;
```
Comment (gvonkoss) [inline client/components/topper/themes/_full-bleed-offset.scss]: shorthand if you want to DELETE MOAR LINES?

File: server/controllers/article-helpers/topper-theme-map.js [hunk 6]
```diff
@@ -11,13 +11,50 @@ const themeImageRatio = {
...
+		const backgroundColour = content.topper.theme === 'full-bleed-offset' ? 'pink' : (content.topper.backgroundColour || 'pink');
+		return {
+			theme: content.topper.theme,
+			template: content.topper.theme === 'full-bleed-offset' ? 'offset' : 'themed',
+			themeImageRatio: themeImageRatio[content.topper.theme],
+			backgroundColour,
+			myFtButtonVariant: myFtButtonVariant(backgroundColour),
+			includesImage: content.topper.theme !== 'full-bleed-text'
+		};
+	} else if (flags.contentPackages && content.containedIn && content.containedIn.length) {
```
Comment (gvonkoss) [inline server/controllers/article-helpers/topper-theme-map.js]: just out of curiosity, if there was no `content.containedIn`, wouldn't `content.containedIn.length` fail anyway?

File: client/components/topper/_main.scss (hunk 2 above)
Comment (adgad) [inline client/components/topper/_main.scss]: Yup, I think we still need the fallback font resizing though. The 18px issue that the ordering was wrong, so the 18px was overriding the 22px in the media query.

File: client/components/topper/_main.scss (hunk 3 above)
Comment (adgad) [inline client/components/topper/_main.scss]: Checked, still works

File: server/controllers/article-helpers/topper-theme-map.js (hunk 6 above)
Comment (adgad) [inline server/controllers/article-helpers/topper-theme-map.js]: It would throw an error `Cannot read property 'length' of null`

**Pull request merged**